js/dealing-with-form.js

```diff
@@ -0,0 +1,32 @@
+const setInactiveStatus = () => {
+  const adForm = document.querySelector('.ad-form');
+  adForm.classList.add('ad-form-disabled');
+  const allFieldsets = document.querySelectorAll('fieldset');
+  allFieldsets.forEach((fieldset) => {
+    fieldset.setAttribute('disabled', 'true');
+  });
+  const mapFilters = document.querySelector('.map__filters');
+  mapFilters.classList.add('ad-form--disabled');
+  const allMapFilter = document.querySelectorAll('.map__filter');
+  allMapFilter.forEach((mapFilter) => {
+    mapFilter.setAttribute('disabled', 'true');
+  });
+};
+setInactiveStatus();
+
+const setActiveStatus = () => {
+  const adForm = document.querySelector('.ad-form');
+  adForm.classList.remove('ad-form-disabled');
+  const allFieldsets = document.querySelectorAll('fieldset');
+  allFieldsets.forEach((fieldset) => {
+    fieldset.removeAttribute('disabled', 'true');
+  });
+  const mapFilters = document.querySelector('.map__filters');
+  mapFilters.classList.remove('ad-form--disabled');
+  const allMapFilter = document.querySelectorAll('.map__filter');
+  allMapFilter.forEach((mapFilter) => {
+    mapFilter.removeAttribute('disabled', 'true');
+  });
+};
+setActiveStatus();
+export {setInactiveStatus, setActiveStatus};
```

Comment:
```suggestion
const setActiveStatus = () => {
  const adForm = document.querySelector('.ad-form');
  adForm.classList.remove('ad-form-disabled');
  
  const allFieldsets = document.querySelectorAll('fieldset');
  allFieldsets.forEach((fieldset) => {
    fieldset.removeAttribute('disabled', 'true');
  });
  
  const mapFilters = document.querySelector('.map__filters');
  mapFilters.classList.remove('ad-form--disabled');
  
  const allMapFilter = document.querySelectorAll('.map__filter');
  allMapFilter.forEach((mapFilter) => {
    mapFilter.removeAttribute('disabled', 'true');
  });
};

setActiveStatus();

export {setInactiveStatus, setActiveStatus};
```

ну и у меня один интересный вопрос. 

А зачем ты все время на момент выполнения функции ищешь ноды? Почему не найти эти ноды ранее, ты все равно с ними же работать будешь. И пусть через область видимости функции имеют доступ к этим переменным.